### PR TITLE
[Validator] Add a Polish translation for invalid charset message

### PR DESCRIPTION
**Pull request by jakzal**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

re #13528

### CODE REVIEW

**Comment by jakzal** [comment]
Fabbot fails as it apparently doesn't speak Polish ;)

**Comment by fabpot** [comment]
Thank you @jakzal.

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 1]
```diff
@@ -278,6 +278,10 @@
                 <source>This value should not be identical to {{ compared_value_type }} {{ compared_value }}.</source>
                 <target>Ta wartość nie powinna być identycznego typu {{ compared_value_type }} oraz wartości {{ compared_value }}.</target>
             </trans-unit>
+            <trans-unit id="73">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Ta wartość nie pasuje do oczekiwanego zestawu znaków {{ charset }}.</target>
```
Comment (eXtreme) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Shouldn't it be "kodowania znaków" instead of "zestawu znaków"?

Comment (jakzal) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: it's the same. The later sounds better to me.

**Pull request merged**